Fix flaky test TestBuildPlanContextDirs

https://travis-ci.org/uber/makisu/builds/457311397
```
--- FAIL: TestBuildPlanContextDirs (0.00s)
	Error Trace:	build_plan_test.go:95
	Error:      	Not equal: 
	            	expected: map[string][]string{"stage1":[]string{"/hello", "/hello2"}}
	            	actual  : map[string][]string{"stage1":[]string{"/hello2", "/hello"}}
	            	
	            	Diff:
	            	--- Expected
	            	+++ Actual
	            	@@ -2,4 +2,4 @@
	            	  (string) (len=6) "stage1": ([]string) (len=2) {
	            	-  (string) (len=6) "/hello",
	            	-  (string) (len=7) "/hello2"
	            	+  (string) (len=7) "/hello2",
	            	+  (string) (len=6) "/hello"
	            	  }
	Test:       	TestBuildPlanContextDirs
```

--- a/lib/builder/build_plan_test.go
+++ b/lib/builder/build_plan_test.go
@@ -92,7 +92,11 @@ func TestBuildPlanContextDirs(t *testing.T) {
 
 	plan, err := NewBuildPlan(ctx, target, cacheMgr, stages, false, false)
 	require.NoError(err)
-	require.Equal(map[string][]string{"stage1": {"/hello", "/hello2"}}, plan.contextDirs)
+	require.Contains(plan.contextDirs, "stage1")
+	require.Len(plan.contextDirs, 1)
+	require.Contains(plan.contextDirs["stage1"], "/hello")
+	require.Contains(plan.contextDirs["stage1"], "/hello2")
+	require.Len(plan.contextDirs["stage1"], 2)
 
 	// Copy from nonexistent stage.
 	from := dockerfile.FromDirectiveFixture("", envImage.String(), "")

